go/service/gregor.go

```diff
@@ -841,21 +841,21 @@ func (g *gregorHandler) newChatMessage(ctx context.Context, m gregor.OutOfBandMe
 		return err
 	}
 
-	var msg keybase1.Message
+	var activity keybase1.ChatActivity
 
-	// TODO: decrypt message from body and fill into plaintext msg
+	// TODO: fill into activity struct and decrypt message if necessary
```

Comment:
hmm.. This also depends on the structure of the OOBM. Do we have that already as well?